Libraries/CMSIS/Device/Maxim/MAX32670/Include/max32670.svd

```diff
@@ -2,7 +2,7 @@
 <device xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" schemaVersion="1.1" xsi:noNamespaceSchemaLocation="svd_schema.xsd">
  <vendor>Maxim-Integrated</vendor>
  <vendorID>Maxim</vendorID>
- <name>MAX32670</name>
+ <name>max32670</name>
```

Comment:
Is lower case consistent with other parts?  Is this field even used anywhere?

Reply:
Yes, I had to make this lowercase for all the parts because ARM's SVDConv utility throws a warning when the part name and the file name don't match. I decided to make the <name> lowercase in the SVD files instead of capitalizing the <part_name>.svd file since the examples already points to the lowercase file version of <part_name>.svd

Reply:
Besides that, this field isn't really significant. I don't think it's used anywhere.